### PR TITLE
Add categories component

### PR DESCRIPTION
**Pull request by Igor333m**
Remove images from assets to static. Add sort to categories.

### CODE REVIEW

File: components/categories.vue [hunk 1]
```diff
@@ -1,20 +1,94 @@
...
+		components: {
+			commonButton
+		},
+		data() {
+			return {
+					categories: [],
+					limit: 10
+			};
+		},
+		mounted() {
```
Comment (Wishoje) [inline components/categories.vue:32]: Ne treba nam mounted zato sto bolje da koristimo server rendering u ovom slucaju.

File: components/categories.vue [hunk 2]
```diff
@@ -1,20 +1,94 @@
 <template>
-	<section class="c-categories">
-		Categories
-	</section>
+	<article class="c-categories">
+		<h2>MOST POPULAR <br />CATEGORIES</h2>
+		<h3>Listen, share and connect your musical world.<br>
+		Go explore the most popular tags.</h3>
+		<section>
```
Comment (Wishoje) [inline components/categories.vue:6]: moze da dodamo v-if="category && category.length > 0" ako ne loaduje categorije zbog nekog razloga

**Review by Wishoje — APPROVED** [review]

File: components/categories.vue [hunk 3]
```diff
@@ -1,20 +1,94 @@
...
+			};
+		},
+		mounted() {
+			this.getCategories();
+		},
+		methods: {
+			async getCategories() {
+				try {
+					const result = await this.$axios.$get('/api/categories');
+					return this.categories = result
```
Comment (Wishoje) [inline components/categories.vue:39]: ne treba return samo "this.categories = result"

File: components/categories.vue [hunk 4]
```diff
@@ -1,20 +1,94 @@
...
+				} catch(error) {
+					console.log('Error :', error);
+				}
+			}
+		},
+		computed: {
+			computedLimit() {
+				(() => {
+					return this.categories = this.categories.sort( (a, b) => {
```
Comment (Wishoje) [inline components/categories.vue:48]: ne treba retrurn ni arrow function
moze ovako:
`this.categories = this.categories.sort( (a, b) => {
					return a.sort - b.sort;
	});
return this.limit ? this.categories.slice(0, this.limit) : this.categories;`

**Comment by Wishoje** [comment]
Merge sam ali popravi ove stvari u nov pr i nek budu po 5 u koloni ne vise
image

**Comment by Igor333m** [comment]
Trebam li da stavim u  created() umesto u mounted()?
 

    On Friday, May 1, 2020, 12:07:50 AM PDT, Wishoje <notifications@github.com> wrote:  
 
 


@Wishoje commented on this pull request.

In components/categories.vue:
> -	name: 'categories'
-}
+	import commonButton from './commonButton';
+
+	export default {
+		name: 'categories',
+		components: {
+			commonButton
+		},
+		data() {
+			return {
+					categories: [],
+					limit: 10
+			};
+		},
+		mounted() {

Ne treba nam mounted zato sto bolje da koristimo server rendering u ovom slucaju.

**Pull request merged**